src/decisionengine/framework/taskmanager/TaskManager.py

```diff
@@ -202,8 +202,8 @@ def __init__(self, name, channel_dict, global_config, de_source_workers=None):
         self.logger = structlog.getLogger(CHANNELLOGGERNAME)
         self.logger = self.logger.bind(module=__name__.split(".")[-1], channel=self.name)
 
-        self.lock = threading.Lock()
         self.broker_url = global_config.get("broker_url", "memory:///")
+        self.broker_url = "redis://localhost:6379/0"
```

Comment:
Should we use this value for the default above?

Reply:
Definitely--I was testing stuff.  Will put this PR back in draft mode for now.

Reply:
Done.